README.md

```diff
@@ -52,6 +52,8 @@ To see the demo version, you can <a href=""> Click here </a>
 
 To get a local copy up and running, follow these steps.
 
+Introduction video about feature of the App. <a href="https://www.loom.com/share/dd62b68443294628abda62698dc78b26"> Click here</a> to check it out.
+
```

Comment:
- Please correct the name of the app written in the readme file please fix this 🙏 
- Also, please confirm that your application is properly deployed, you can deploy it on render, netlify. Please fix this 😄

Reply:
Well resolved 👏🏾